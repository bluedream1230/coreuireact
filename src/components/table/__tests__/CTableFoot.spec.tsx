import * as React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { CTableFoot } from '../../../index'

test('loads and displays CTableFoot component', async () => {
  const { container } = render(<CTableFoot/>)
  expect(container).toMatchSnapshot()
})

test('CTableFoot customize', async () => {
    const { container } = render(
        <CTableFoot
            className="bazinga"
            color="info"
        >
            Test
        </CTableFoot>
    )
    expect(container).toMatchSnapshot()
    expect(container.firstChild).toHaveClass('table-info')
    expect(container.firstChild).toHaveClass('bazinga')
    expect(container.firstChild).toHaveTextContent('Test')
})