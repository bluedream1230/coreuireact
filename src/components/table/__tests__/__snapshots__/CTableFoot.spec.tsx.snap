// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CTableFoot customize 1`] = `
<div>
  <tfoot
    class="table-info bazinga"
  >
    Test
  </tfoot>
</div>
`;

exports[`loads and displays CTableFoot component 1`] = `
<div>
  <tfoot />
</div>
`;
